t/03_tools.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use warnings;
+use Test::More tests => 3;
+use Test::Alien;
+use Alien::libtiff;
+use File::Temp;
+
+alien_ok 'Alien::libtiff';
+
+SKIP: {
```

Comment:
It is better to check the install type from the Alien itself.

```perl
skip "The default system install on Windows doesn't include libtiff tools",
      2 unless $ENV{ALIEN_INSTALL_TYPE} eq 'share';
```

Reply:
Yup. Thanks. Implemented.